catalog/spatial/kml/spatial-kml-transformer/src/main/java/org/codice/ddf/spatial/kml/transformer/KMLTransformerImpl.java

```diff
@@ -208,6 +221,91 @@ protected Placemark performDefaultTransformation(Metacard entry)
     return kmlPlacemark;
   }
 
+  private void setExtendedData(Placemark placemark, Metacard metacard) {
+    final ExtendedData extendedData = new ExtendedData();
+
+    final Set<AttributeDescriptor> attributeDescriptors =
+        metacard.getMetacardType().getAttributeDescriptors();
+
+    for (AttributeDescriptor attributeDescriptor : attributeDescriptors) {
+      final String attributeName = attributeDescriptor.getName();
+      final Attribute attribute = metacard.getAttribute(attributeName);
+      if (attribute != null) {
+        String attributeValue =
+            convertAttribute(attribute, attributeDescriptor)
+                .toString()
+                .replace("[", "")
+                .replace("]", "");
+        final Data data = getData(attributeName, attributeValue);
+        extendedData.addToData(data);
+      }
+    }
+    placemark.setExtendedData(extendedData);
+  }
+
+  private Data getData(String attributeAlias, String attributeValue) {
+    final Data data = new Data(attributeValue);
+    data.setName(attributeAlias);
+    return data;
+  }
+
+  private static Object convertAttribute(Attribute attribute, AttributeDescriptor descriptor) {
+
+    if (descriptor.isMultiValued()) {
+      List<Object> values = new ArrayList<>();
+      for (Serializable value : attribute.getValues()) {
+        values.add(
+            convertValue(attribute.getName(), value, descriptor.getType().getAttributeFormat()));
+      }
+      return values;
+    } else {
+      return convertValue(
+          attribute.getName(), attribute.getValue(), descriptor.getType().getAttributeFormat());
+    }
+  }
+
+  private static Object convertValue(
+      String name, Serializable value, AttributeType.AttributeFormat format) {
+    if (value == null) {
+      return null;
+    }
+
+    switch (format) {
+      case DATE:
+        if (!(value instanceof Date)) {
+          LOGGER.debug(
+              "Dropping attribute date value {} for {} because it isn't a Date object.",
+              value,
+              name);
+          return null;
+        }
+        // Creating date format instance each time is inefficient, however
+        // it is not a threadsafe class so we are not able to put it in a static
+        // class variable. If this proves to be a slowdown this class should be refactored
+        // such that we don't need this method to be static.
+        SimpleDateFormat dateFormat = new SimpleDateFormat(ISO_8601_DATE_FORMAT);
+        dateFormat.setTimeZone(TimeZone.getTimeZone("GMT"));
```

Comment:
❓ I know that UTC and GMT but should we stick to UTC for consistency? 

`TimeZone.getTimeZone("UTC")`